shared/src/commonTest/kotlin/com/innova/innovaPoolingLibrary/VehicleNumberValidationTest.kt

```diff
@@ -0,0 +1,49 @@
+package com.innova.innovaPoolingLibrary
+
+import com.innova.innovaPoolingLibrary.utils.regexUtils.RegexUtils
+import kotlin.test.Test
+import kotlin.test.assertFalse
+import kotlin.test.assertTrue
+
+class VehicleNumberValidationTest {
```

Comment:
write a test case whether white spaces are accepting or not

Reply:
done